### PR TITLE
FEAT: add xorbits experimental

### PR DESCRIPTION
**Pull request by Hank0626**
## What do these changes do?
- add xorbits experimental with text deduplication

Fixes #xxxx

## Check code requirements

- [x] tests added / passed (if needed)
- [x] Ensure all linting tests pass

### CODE REVIEW

File: python/xorbits/experimental/tests/test_text_dedup.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+import pandas as pd
+import pytest
+
+from ..._mars.dataframe.datasource.dataframe import from_pandas as from_pandas_df
+from ...experimental import dedup
+
+
+def test_dedup_execute(setup):
+    words = list("abcdefghijklmnopqrstuvwxyz")
+    df_raw = pd.DataFrame(
```
Comment (aresnow1) [inline python/xorbits/experimental/tests/test_text_dedup.py:25]: Better use xorbits.pandas to create a DataFrame.

Comment (Hank0626) [reply python/xorbits/experimental/tests/test_text_dedup.py:25]: done

**Review by aresnow1 — APPROVED** [review]
LGTM

File: python/xorbits/_mars/dataframe/base/__init__.py [hunk 2]
```diff
@@ -86,7 +85,6 @@ def _install():
         setattr(
             t, "__delitem__", lambda df, items: df_drop(df, items, axis=1, inplace=True)
         )
-        setattr(t, "dedup", df_dedup)
```
Comment (codingl2k1) [inline python/xorbits/_mars/dataframe/base/__init__.py:89]: The `dedup()` method is unregistered from mars DataFrame. How can I use the dedup feature?

Comment (Hank0626) [reply python/xorbits/_mars/dataframe/base/__init__.py:89]:
```python
from xorbits.experimental import dedup

df = # your data

res = dedup(df, col="text")
```

Comment (Hank0626) [reply python/xorbits/_mars/dataframe/base/__init__.py:89]: dedup is not the feature of dataframe anymore, it belongs to a new collection named experimental

File: python/xorbits/experimental/dedup.py [hunk 3]
```diff
@@ -472,9 +475,7 @@ def df_dedup(
     ...         * 2,
     ...     }
     ... )
-    >>> res = df.dedup(col="text")
-    >>> res.execute()
-
+    >>> res = dedup(df, col="text")
```
Comment (codingl2k1) [inline python/xorbits/experimental/dedup.py:478]: Got it. The usage is just like the `pd.concat`.

```python
from xorbits.experimental import dedup
dedup(df)
```

**Review by codingl2k1 — APPROVED** [review]
LGTM

File: python/xorbits/_mars/dataframe/base/__init__.py (hunk 2 above)
Comment (codingl2k1) [reply python/xorbits/_mars/dataframe/base/__init__.py:89]: It's our extension to pandas DataFrame. That makes sense.

**Pull request merged**